README.md

```diff
@@ -11,46 +11,51 @@
 - [unzip](http://infozip.sourceforge.net/UnZip.html)
 - [jq](https://stedolan.github.io/jq/) (only for updating the release data)
 
+_Besides bash (the shell used by the maintainer) this plugin **should** work on Fish, Zsh and Xonsh_
```

Comment:
The tests in the GitHub workflows should show that it's working. 😉

Reply:
True, but I wanted to make clear what was guaranteed. Getting the Xonsh part to work properly took me quite some time, which annoyed me a bit. That's what made me add this line.